spec/controllers/projects_controller_spec.rb

```diff
@@ -0,0 +1,108 @@
+require 'rails_helper'
+
+RSpec.describe ProjectsController, type: :controller do
+  before(:each) do
+    @user = User.create!(name: 'midd', email: 'user@example.org', password: 'very-secret')
+    @user.role = :developer
+    @user.save!
+  end
+  context 'get' do
+    it 'returns a successful response' do
+      get :index
+      expect(response).to be_successful
+    end
+
+    it 'assigns @projects' do
+      @project = Project.create(title: 'Project1',
```

Comment:
Can we move user and project creation in their own method?
Maybe it user creation would fit in before block?